client/extensions/woocommerce/app/dashboard/pre-setup-view.js

```diff
@@ -75,16 +130,31 @@ class PreSetupView extends Component {
 	}
 }
 
-function mapStateToProps( /* state */ ) {
+function mapStateToProps( state ) {
+	const site = getSelectedSiteWithFallback( state );
+	let name = '';
+	if ( site ) {
+		name = getSiteTitle( state, site.ID );
```

Comment:
You can reduce these 4 lines down to just one: `const name = site && getSiteTitle( state, site.ID ) || ''`.